client/components/README.md

```diff
@@ -1,8 +1,10 @@
 Components
 ==========
 
-_Read the [documentation on components](https://github.com/Automattic/wp-calypso/blob/master/docs/components.md) for more in-depth information._
+Components are React components used for composing the UI of Calypso.
```

Comment:
Should we add a few notes on how they behave more as UI primitives?

For example we'd usually be able to write a component as a functional stateless component (provided that folks didn't need to keep track of refs).